### PR TITLE
Fix/minor fixes

### PR DESCRIPTION
**Pull request by Roblinde**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -56,7 +57,8 @@ def set_session_permanency():
 
 
 # Register Markdown engine
-Misaka(app)
+# Misaka(app, None, hard_wrap=True)
+app.add_template_filter(markdown)
```
Comment (dborsatto) [inline app.py]: I think you can remove `from flask_mikasa import Mikasa` and the overall dependency now, right?

File: views/layout.html [hunk 2]
```diff
@@ -217,22 +217,7 @@
...
-            <svg>
-              <use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#twitter"></use>
-            </svg>
-          </a>
-          <a href="https://github.com/contentful" target="_blank" rel="noopener">
-            <svg>
-              <use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#github"></use>
-            </svg>
-          </a>
+          <p><a href="https://www.facebook.com/contentful/" target="_blank" rel="noopener"><svg><use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#facebook"></use></svg></a><a href="https://twitter.com/contentful" target="_blank" rel="noopener"><svg><use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#twitter"></use></svg></a><a href="https://github.com/contentful" target="_blank" rel="noopener"><svg><use xlink:href="{{ url_for('static', filename="icons/icons.svg") }}#github"></use></svg></a>
```
Comment (dborsatto) [inline views/layout.html]: I think you don't need to remove the whitespace between 2 different `a` tags, just the one _inside_.
So you can still have 3 `a` tags on 3 different lines:
``` html
<p>
  <a>...</a>
  <a>...</a>
  <a>...</a>
</p>
```

File: lib/markdown.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+    :param text: String to be transformed.
+    :return: Transformed html string.
+
+    Usage:
+
+        {{ 'Some *markdown*'|markdown }}
+        "Some <strong>markdown</strong>"
+    """
+    print(text)
```
Comment (dborsatto) [inline lib/markdown.py]: Is this a leftover for debugging purposes?

File: app.py (hunk 1 above)
Comment (Roblinde) [reply app.py]: had forgot to save the app.py file, sigh. Update inc.

**Pull request merged**